lib/quests/json/piggy_nibbler.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "Q11",
+  "title": "Meat Galore",
+  "description": "Go nibble 6 pigs",
+  "requirements": [
+    {
+      "id": "Q11R2",
+      "type": "counter_unique",
```

Comment:
counter_unique is meant to only increase the count for unique evenTypes. For example, in the tree petter quest, we want to only give the player credit if they are petting different kinds of trees. So our eventType is treePet.* which means it will match things like "treePetBubble" and "treePetFruit". But since the type is "counter_unique" we will only increase the count if the player has never before had a certain type of event.

In this piggy quest case, I think you want to use "counter" for the type so that it will increase the count each time they nibble a piggy. I also think you can just change eventType to "piggyNibble" to match my above comment in the piggy.dart file.